notes/ltc-transfer-penalty.vbs

```diff
@@ -94,17 +98,10 @@ EMReadScreen app_year, 2, 12, 39
 date_of_application = (app_month & "/" & app_date & "/" & app_year)
 
 'navigating to the Uncompensated Transfer Calculation in the ELIG/HC person test
-Call navigate_to_MAXIS_screen ("elig", "HC__")
-'selects MEMB 01
-EMWriteScreen "x", 8, 26
-transmit
-'selects Person test
-EMWriteScreen "x", 7, 17
-transmit
-'selects Uncompensated transfer calculation
-EMWriteScreen "x", 18, 3
-transmit
-'reading information from Uncompensated Transfer Calculation
+Call navigate_to_MAXIS_screen ("ELIG", "HC__")        'selects MEMB 01
+Call write_value_and_transmit("X", 8, 26)             'selects Person test
+Call write_value_and_transmit("X", 7, 17)             'selects Uncompensated transfer calculation
+Call write_value_and_transmit("X", 18, 3)             'reading information from Uncompensated Transfer Calculation
 EMReadScreen row_1, 71, 5, 6
 EMReadScreen row_3, 71, 7, 6
 EMReadScreen row_4, 71, 8, 6
```

Comment:
I know this isn't your code, but....
Line 120 and Line 133 - these variables won't be "" since they are reading underscore. I recommend adding handling that looks like If trim(transfer_amount <> "__ __ __") then ....
Then if it's blank on the panel it will be blank in the dialog vs. underscore that are formatted with the /.